app/api/chat/[id]/message/route.ts

```diff
@@ -1,73 +1,51 @@
-import { NextResponse } from "next/server";
-import { z, ZodError } from "zod";
-import formatZodError from "@/components/format-zod-error";
+import { z } from "zod";
+import { withApiWrapper } from "@/utils/api-wrapper";
 import { addInboxMessage } from "@/server";
 import { NotFoundError } from "@/server/errors";
 import { ChatSessionsTable, ContactsTable } from "@/db/schema";
 import { eq } from "drizzle-orm";
 import { getDb } from "@/db/connection";
 
-export async function POST(
+async function chatMessageHandler(
+  body: unknown,
   request: Request,
-  { params }: { params: Promise<{ id: string }> }
+  context?: unknown
```

Comment:
instead of passing in a `context` into the handler, update `withApiWrapper` to await the `context` if it exists and spread the params into the body.